packages/waku/src/lib/builder/build.ts

```diff
@@ -774,10 +774,4 @@ export async function build(options: {
 
   platformObject.buildOptions.unstable_phase = 'buildDeploy';
   await buildDeploy(rootDir, config);
-  delete platformObject.buildOptions.unstable_phase;
-
-  await appendFile(
-    distEntriesFile,
-    `export const buildData = ${JSON.stringify(platformObject.buildData)};`,
-  );
```

Comment:
Please keep this because this is required even without deploy plugins.
Only change cloudflare deploy plugin. I don't know the future, but my understanding is only cloudflare has special requirement for the dist folder structure.

Reply:
Wouldn't it be the same previously?
I'm fine to add a check:
```js
if (existsSync(distEntriesFile)) {
  await appendFile(
    distEntriesFile,
    `export const buildData = ${JSON.stringify(platformObject.buildData)};`,
  );
}
```

Reply:
Ah... good idea.

Reply:
@dai-shi OK great. I amended this PR.